src/test/java/spoon/test/trycatch/TryCatchTest.java

```diff
@@ -421,15 +422,18 @@ public void testTryWithVariableAsResource() {
 		factory.getEnvironment().setComplianceLevel(9);
 
 		// contract: a try with resource is modeled with CtTryWithResource
-		CtTryWithResource tryStmt = factory.Code().createCodeSnippetStatement("" +
+		CtTryWithResource tryStmt = factory.Code().createCodeSnippetStatement("class A { public void m() throws Exception {" +
 				"java.lang.AutoCloseable resource = null;" +
-				"try(resource){}"
-			).compile();
+				"try(resource){}}}"
+			).compile().getElements(new TypeFilter<>(CtTryWithResource.class)).get(0);
         List<CtResource<?>> resources = tryStmt.getResources();
         assertEquals(1, resources.size());
 		final CtResource<?> ctResource = resources.get(0);
-		assertTrue(ctResource instanceof CtVariableRead);
-        assertEquals("resource", ((CtVariableRead<?>) ctResource).getVariable().getSimpleName());
+		assertTrue(ctResource instanceof CtVariable);
+        assertEquals("resource", ((CtVariable<?>) ctResource).getSimpleName());
```

Comment:
Convert the spaces to tabs here. We are not super strict with formatting in testcases but a method should either use spaces or tabs, not both